README.md

```diff
@@ -303,24 +303,24 @@ let subKey2 = sodium.keyDerivation.derive(secretKey: secretKey,
 
 ```swift
 let sodium = Sodium()
-var dataToZero = "Message".bytes
+var dataToZero = "Message".data(using:.utf8)!
 sodium.utils.zero(&dataToZero)
 ```
 
 ### Constant-time comparison
 
 ```swift
 let sodium = Sodium()
-let secret1 = "Secret key".bytes
-let secret2 = "Secret key".bytes
+let secret1 = BytesContainer("Secret key")
```

Comment:
I wasn't sure how we want to handle implementing constant time stuff with the protocol, so these methods are only implemented over `BytesContainer` so far. Because of this the inputs need to be `BytesContainer`s at present.